recipe/meta.yaml

```diff
@@ -16,13 +16,15 @@ build:
 requirements:
     build:
         - python
+        - setuptools
```

Comment:
Try `pip` here and update the script line to `script: python -m pip install --no-deps --ignore-installed .`

See https://conda-forge.org/docs/meta.html#use-pip

Reply:
@ocefpaf Got it, finally :+1: